p2p/server.go

```diff
@@ -854,13 +854,16 @@ func (srv *Server) listenLoop() {
 		<-slots
 
 		var (
-			fd  net.Conn
-			err error
+			fd        net.Conn
+			err       error
+			tmpErrors = uint64(0)
 		)
 		for {
 			fd, err = srv.listener.Accept()
-			if netutil.IsTemporaryError(err) {
+			if netutil.IsTemporaryError(err) && tmpErrors < 200 {
```

Comment:
I'm not sure I like this limit. The delay is OK, but the limit can be a problem: if the node runs out of file descriptors temporarily (i.e. not during shutdown), listening will not recover once it's over the threshold.

Reply:
The solution could be to simply make the delay longer and remove the limit.

Reply:
Ok, what's your referred delay? How about starting at `10ms`, then adding doubling the delay but cap the delay at 2 seconds  ?

Reply:
I think 2s is too much. It might be OK to just use a 200ms static delay here.

Reply:
I agree, delay-wise, however, spitting out 5 lines of logs per second seems excessive, so we should still limit the log output to max one line per second, IMO

Reply:
Ok, fixed, PTAL